frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -49,14 +52,14 @@ export interface FeatureFlagLogicProps {
     id: number | 'new'
 }
 
-export const featureFlagLogic = kea<featureFlagLogicType<FeatureFlagLogicProps>>({
-    path: (key) => ['scenes', 'feature-flags', 'featureFlagLogic', key],
-    props: {} as FeatureFlagLogicProps,
-    key: ({ id }) => id ?? 'new',
-    connect: {
+export const featureFlagLogic = kea<featureFlagLogicType<FeatureFlagLogicProps>>([
+    path(['scenes', 'feature-flags', 'featureFlagLogic']),
+    props({} as FeatureFlagLogicProps),
```

Comment:
Would be pretty cool ergonomics-wise to be able to pass the type of props as `props<FeatureFlagLogicProps>()` now (maybe? I'm not sure if that'd actually work with the architecture of those new functions)